ui/tests/acceptance/auth-test.js

```diff
@@ -69,22 +69,25 @@ module('Acceptance | auth', function (hooks) {
       }
       await component.login();
       const lastRequest = this.server.passthroughRequests[this.server.passthroughRequests.length - 1];
-      let body = JSON.parse(lastRequest.requestBody);
-      // Note: x-vault-token used to be lowercase prior to upgrade
+      const body = JSON.parse(lastRequest.requestBody);
+
+      let keys;
+      let included;
       if (backend.type === 'token') {
-        assert.ok(
-          Object.keys(lastRequest.requestHeaders).includes('X-Vault-Token'),
-          'token uses vault token header'
-        );
+        keys = lastRequest.requestHeaders;
+        included = 'X-Vault-Token';
```

Comment:
for whatever reason when we upgrade to ember data's next major version we need to lower case this.